### ISSUE TITLE
Compiling on linux fails

### ISSUE THREAD

**Issue by wipu** [asb2m10/dexed#218]
Hello.

In December I was able to compile dexed version 0.9.4. But now I get some "cannot bind packed field" errors from juce. It seems this is a gcc version issue.

So I tried compiling from the latest commit, bdcc7d257867f5cf87d392fc6079322f0980c05e, by running scripts/build.sh. It downloads some juce files but then fails:

> inflating: JUCE/extras/Projucer/Projucer.jucer  
> Re-saving file: /tmp/dexed-bdcc7d257867f5cf87d392fc6079322f0980c05e.work/Dexed.jucer
> Error when saving: At least one of your JUCE module paths is invalid!
> Please go to the Modules settings page and ensure each path points to the correct JUCE modules folder.
> ./scripts/build-lin.sh: line 4: cd: Builds/Linux: No such file or directory
...

I cannot find any documentation for building. So what should I do?

**Comment by encore64** [comment]
@wipu With the recent pull request merge, the build process is different now. There are instructions in the README.md [here](https://github.com/asb2m10/dexed#all-oses) on how to build it.

You still need some (all?) packages installed first to successfully build it though, see here: https://github.com/asb2m10/dexed/wiki/Linux-Builds

Using this, I can confirm building bdcc7d2 works in Ubuntu 19.10.

**Comment by wipu** [comment]
Oh, somehow I missed the readme. While trying out different versions and ways of getting them (git clone vs release download) I managed to only read some old readme. Doh, sorry to bother you.

Yes. I was now able to compile the standalone version, which is the only supported format nowadays :(

It's good that I managed to build a working vst for me in December, before things got broken. Hopefully it will work until a new version becomes buildable again.

Closing the ticket.
